src/scripts/Selector/SelectorTable.js

```diff
@@ -264,34 +264,59 @@ export default class SelectorTable extends Selector {
 
 	/**
 	 * Extract table header column info from html
-	 * @param headerRowSelector
-	 * @param html
-	 * @param verticalTable
+	 * @param $headerRow
+	 * @param columns
+	 * @param tr_num
+	 * @param col_num
+	 * @param name
+	 * @param k
+	 * @param limit_col
 	 */
+	static columnsMaker($headerRow, columns, tr_num, col_num, name, k, limit_col) {
```

Comment:
We need somehow refactor this function and hide local parameters like tr_num, col_num, name, k, limit_col and make this function generating columns object.

Reply:
In my opinion, these changes will be difficult to implement and this version easy to understand

Reply:
I spent some time trying to understand that function but failed to understand it still. I am asking you to expirement with this and make its interface as i recommended.